ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueConsumption.scala

```diff
@@ -221,7 +221,8 @@ object KeyValueConsumption {
       BaseEncoding.base16.encode(entryId.toByteArray)
     )
     Update.TransactionAccepted(
-      optSubmitterInfo = Some(parseSubmitterInfo(txEntry.getSubmitterInfo)),
+      optSubmitterInfo =
```

Comment:
Can you add a unit test case for this, please?

Reply:
Added.